### PR TITLE
Portfolio: Finish mobile version

### PR DESCRIPTION
**Pull request by malikhaiderkhan**
I created cards in "works" section and also created "About Myself" section and stick to the design as much as possible.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -74,6 +74,200 @@ nav a:hover {
...
   cursor: pointer;
 }
+
+.portfolio {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  margin: 70px 3.2%;
+}
+
```
Comment (shella12) [inline style.css:98]: - [x] The background color of `work` section in the template is `#E5E5E5` and according to the requirement we should use exact same colors from[ template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=uPVexXRUnbDHuZTG-0). Kindly change the background color of work section to  `#E5E5E5`. 

| Figma design | Your design |
|--------------|--------------|
|Screenshot 2023-04-29 at 1 55 23 PM|Screenshot 2023-04-29 at 1 55 51 PM|

File: style.css [hunk 2]
```diff
@@ -74,6 +74,200 @@ nav a:hover {
...
+.about {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  border-top-right-radius: 60px;
+  padding: 114px 6.5%;
+  position: relative;
+  z-index: 1;
+}
```
Comment (shella12) [inline style.css:205]: - [x] The background color of `about` section in the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=uPVexXRUnbDHuZTG-0) is has a rounded-right corner and according to the requirement your design should be as close to the template as possible.  Kindly make right corner of about  section `rounded. 

| Figma design | Your design |
|--------------|--------------|
|Screenshot 2023-04-29 at 1 55 35 PM|Screenshot 2023-04-29 at 1 58 55 PM|

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @malikhaiderkhan, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- All linters passed ✅ 
- Professional README.md ✅   
- Clean code ✅ 
- Great design implementation ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (malikhaiderkhan) [reply style.css:98]: I done this task by adding background color as required.

File: style.css (hunk 2 above)
Comment (malikhaiderkhan) [reply style.css:205]: Also added rounded shape as required by reviewer.

**Review by amon-cofie — APPROVED** [review]
Hi @malikhaiderkhan, 👋 

Your project is complete and very impressive 💯!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

🚀 You implemented the changes requested by the previous reviewer very well
🚀 You have a good commit history
🚀 You used GitHub flow properly
🚀 You have no linter errors


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**